app/logic/solana_logic.rb

```diff
@@ -52,6 +52,27 @@ def epoch_get
         slot_index: epoch_json['slotIndex'],
         slots_in_epoch: epoch_json['slotsInEpoch']
       )
+      begin
+        EpochWallClock.create!(
```

Comment:
I did some simulations and the speed is very similar to raw INSERT IGNORE query. In my opinion this solution is a bit more secure.